spec/guardtime_driver_spec.rb

```diff
@@ -1,27 +1,43 @@
 require "spec_helper"
 require "date"
 
-RSpec.shared_examples "Guardtime" do |gt_driver|
-  username = ENV['GUARDTIME_user']
-  password = ENV['GUARDTIME_password']
-  now = DateTime.now
+username = ENV['GUARDTIME_user']
+password = ENV['GUARDTIME_password']
+
+use_vcr = !username
+tag = use_vcr ? "(VCR)" : ""
+
+username ||= "username"
+password ||= "password"
+
+if use_vcr
+  print "Using VCR for Guardtime tests ...\n"
+  VCR.configure do |config|
+    config.cassette_library_dir = "spec/vcr_cassettes"
+    config.allow_http_connections_when_no_cassette = true
+    config.hook_into :webmock
+    config.configure_rspec_metadata!
+  end
```

Comment:
Normally there would be something here to filter out the user and password from the VCR cassettes using https://relishapp.com/vcr/vcr/docs/configuration/filter-sensitive-data, but I can't see anything in there. However, VCR is also not recording the HTTP Basic headers in the cassettes, so perhaps it's filtering them automatically?

Reply:
I don't think there anything to change here, as I suspect it's automatic, but I just wanted to raise it so we know we're safe.

Reply:
I took out the Basic auth headers by hand before committing the yml files.  I missed that config item - I'll have a look.